versions.txt

```diff
@@ -2,16 +2,16 @@
 # by default with the OpenTelemetry Operator. This would usually be the latest
 # stable OpenTelemetry version. When you update this file, make sure to update the
 # the docs as well.
-opentelemetry-collector=0.117.0
+opentelemetry-collector=0.118.0
 
 # Represents the current release of the OpenTelemetry Operator.
-operator=0.117.0
+operator=0.118.0
```

Comment:
There is a new go instrumentation https://github.com/open-telemetry/opentelemetry-go-instrumentation/releases